wazo_calld/plugins/parking_lots/schemas.py

```diff
@@ -0,0 +1,92 @@
+# Copyright 2024 The Wazo Authors (see the AUTHORS file)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import annotations
+
+from datetime import datetime, timedelta, timezone
+from typing import TYPE_CHECKING
+
+from marshmallow import EXCLUDE, Schema, post_dump, pre_dump
+from marshmallow.fields import Method
+from marshmallow.validate import Predicate
+from xivo.mallow.fields import Integer, List, Nested, String
+
+if TYPE_CHECKING:
+    from .dataclasses_ import AsteriskParkedCall, ConfdParkingLot
+
+
+class _Base(Schema):
+    class Meta:
+        ordered = True
+        unknown = EXCLUDE
+
+
+class ParkingLotSchema(_Base):
+    name = String(dump_only=True)
+    exten = String(dump_only=True, default='')
```

Comment:
Why duplicate these values from wazo-confd resource?
I don't have strong opinion for name, but the `exten` is only used for the desktop phone context, not in the API